src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -57,7 +57,7 @@ public boolean isFull() {
      */
     protected int peekaboo() {
         if (callCheck())
-            return INT;
+            return EMPTY;
```

Comment:
This constant should be different (separate) than the one in peekaboo and countOut methods. (tiny.cc/998muz)